### ISSUE TITLE
Build fails with Poppler 0.86: no viable conversion from 'const std::string' to 'const pdf::String *'

### ISSUE THREAD

**Issue by dbevans** [jwilk/pdf2djvu#142]
Building using

MacPorts 2.6.2
macOS 10.15.3 19D76 (Catalina)
Xcode 11.3.1 11C504 
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin19.3.0
Thread model: posix

```
/usr/bin/clang++ -pipe -Os -DHAVE_SYS_TYPES_H=1 -stdlib=libc++ -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -arch x86_64 -Wall -Wempty-body -Werror=narrowing -Werror=overloaded-virtual  -I/opt/local/include/poppler -I/opt/local/include/GraphicsMagick  -I/usr/include/uuid/uuid.h -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -std=gnu++11  -c -o pdf2djvu.o pdf2djvu.cc
pdf2djvu.cc:486:32: error: no viable conversion from 'const std::string' (aka 'const basic_string<char, char_traits<char>, allocator<char> >') to 'const pdf::String *' (aka 'const GooString *')
      uri += pdf::get_c_string(dynamic_cast<pdf::link::URI*>(link_action)->getURI());
                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:876:5: note: candidate function
    operator __self_view() const _NOEXCEPT { return __self_view(data(), size()); }
    ^
./pdf-backend.hh:439:48: note: passing argument to parameter 'str' here
  const char * get_c_string(const pdf::String *str);
                                               ^
pdf2djvu.cc:725:29: error: no viable conversion from 'std::unique_ptr<LinkAction>' to 'std::__1::unique_ptr<LinkAction, std::__1::default_delete<LinkAction> >::pointer' (aka 'LinkAction *')
          link_action.reset(pdf::link::Action::parseDest(&destination));
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/memory:2648:22: note: passing argument to parameter '__p' here
  void reset(pointer __p = pointer()) _NOEXCEPT {
                     ^
pdf2djvu.cc:727:29: error: no viable conversion from 'std::unique_ptr<LinkAction>' to 'std::__1::unique_ptr<LinkAction, std::__1::default_delete<LinkAction> >::pointer' (aka 'LinkAction *')
          link_action.reset(pdf::link::Action::parseAction(&destination));
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/memory:2648:22: note: passing argument to parameter '__p' here
  void reset(pointer __p = pointer()) _NOEXCEPT {
                     ^
3 errors generated.
make: *** [pdf2djvu.o] Error 1

```

**Comment by jwilk** [comment]
Fixed in b28c2007ca62fd5bae2c1d64b549ae16e849bb9e and d3bb5259180383de2de2f6456e07e79e3972254c. I'll make a new release later today.

BTW, beware that Poppler 0.86.0 is seriously broken; please upgrade to 0.86.1.

**Comment by jwilk** [comment]
Fixed in [0.9.17](https://groups.io/g/pdf2djvu/message/129).
